pgt/src/PGT/Output/Test/Result.hs

```diff
@@ -67,17 +66,18 @@ instance Render Record where
     = unlines $ convert title <| items
 
 data RowResults = RowResults
-  { columns :: Text
-  , rows    :: NonEmpty Text
+  { columns  :: Text
+  , rowCount :: RowCount
```

Comment:
__suggestion: (nitpick)__ Given the datatype is "row results" having a field named "row count" is somewhat redundant. If the field was `count` I think it would be clear what it refers to.